akommons/src/main/java/com/dewarder/akommons/content/res/TypedArray.kt

```diff
@@ -37,4 +43,16 @@ inline fun <R> TypedArray.use(block: (TypedArray) -> R): R {
             recycle()
         }
     }
-}
\ No newline at end of file
+}
+
+inline fun <R> Context.useStyledAttributes(@StyleableRes attrs: IntArray, block: (TypedArray) -> R): R
+    = obtainStyledAttributes(attrs).use(block)
+
+inline fun <R> Context.useStyledAttributes(@StyleRes resid: Int, @StyleableRes attrs: IntArray, block: (TypedArray) -> R): R
```

Comment:
Please, format all parameters of functions according to [Coding Conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting) in case if it doesn't fit 80 symbols limit.